tensorflow/core/kernels/mkl_conv_ops.h

```diff
@@ -351,15 +368,23 @@ class MklDnnConvUtil {
 
     // Output batch is same as input batch.
     int out_batch = GetTensorDim(input_shape, data_format_, 'N');
-
-    // Output depth is same as last dimension for filter.
-    int out_depth = filter_shape.dim_size(isConv2D ? 3 : 4);
+    int out_depth;
+    
+    // TODO add support for 3-D Depthwise
+
+    // Output depth is same as last dimension for filters for regular 
+    // convolutions. For depthwise it is in_depth*channel_muliplier 
```

Comment:
Please correct the typo and add spaces for readability.
`in_depth*channel_muliplier` -> `in_depth * channel_multiplier`

This file also haven't mentioned `channel_multiplier` before, so please also explain that this means the `out_depth` of the filter shape.